setup.py

```diff
@@ -138,23 +138,25 @@ def get_branch():
         "dessia_common.schemas"
     ],
     install_requires=[
-        "orjson>=3.8.0",
-        "typeguard",
+        "cma",
+        "dectree",
+        "docx",
+        "humanize",
+        "matplotlib",
+        "mypy_extensions",
```

Comment:
Maybe we could remove mypy_extensions in the future, as we state that python version is <3.9. It was necessary for python <3.8